qiskit/test/mock/backends/almaden/conf_almaden.json

```diff
@@ -1 +1 @@
 "jq7q12*Sm7*Sp12", "jq8q9*Sp8*Sm9", "jq8q9*Sm8*Sp9", "jq15q16*Sp15*Sm16", "jq15q16*Sm15*Sp16", "jq1q6*Sp1*Sm6", "jq1q6*Sm1*Sp6", "jq18q19*Sp18*Sm19", "jq18q19*Sm18*Sp19", "jq1q2*Sp1*Sm2", "jq1q2*Sm1*Sp2", "jq16q17*Sp16*Sm17", "jq16q17*Sm16*Sp17", "jq6q7*Sp6*Sm7", "jq6q7*Sm6*Sp7", "jq12q13*Sp12*Sm13", "jq12q13*Sm12*Sp13", "jq3q4*Sp3*Sm4", "jq3q4*Sm3*Sp4", "jq9q14*Sp9*Sm14", "jq9q14*Sm9*Sp14", "jq2q3*Sp2*Sm3", "jq2q3*Sm2*Sp3", "jq11q12*Sp11*Sm12", "jq11q12*Sm11*Sp12", "jq0q1*Sp0*Sm1", "jq0q1*Sm0*Sp1", "jq17q18*Sp17*Sm18", "jq17q18*Sm17*Sp18", "jq5q10*Sp5*Sm10", "jq5q10*Sm5*Sp10", "jq13q14*Sp13*Sm14", "jq13q14*Sm13*Sp14", "jq3q8*Sp3*Sm8", "jq3q8*Sm3*Sp8", "jq13q18*Sp13*Sm18", "jq13q18*Sm13*Sp18", "jq7q8*Sp7*Sm8", "jq7q8*Sm7*Sp8", "omegad1*X0||U0", "omegad0*X1||U1", "omegad6*X1||U3", "omegad2*X1||U2", "omegad1*X2||U4", "omegad3*X2||U5", "omegad4*X3||U7", "omegad2*X3||U6", "omegad8*X3||U8", "omegad3*X4||U9", "omegad10*X5||U11", "omegad6*X5||U10", "omegad7*X6||U14", "omegad1*X6||U12", "omegad5*X6||U13", "omegad6*X7||U15", "omegad8*X7||U16", "omegad12*X7||U17", "omegad7*X8||U19", "omegad3*X8||U18", "omegad9*X8||U20", "omegad8*X9||U21", "omegad14*X9||U22", "omegad11*X10||U24", "omegad5*X10||U23", "omegad10*X11||U25", "omegad12*X11||U26", "omegad16*X11||U27", "omegad7*X12||U28", "omegad13*X12||U30", "omegad11*X12||U29", "omegad12*X13||U31", "omegad14*X13||U32", "omegad18*X13||U33", "omegad9*X14||U34", "omegad13*X14||U35", "omegad16*X15||U36", "omegad15*X16||U38", "omegad17*X16||U39", "omegad11*X16||U37", "omegad18*X17||U41", "omegad16*X17||U40", "omegad13*X18||U42", "omegad17*X18||U43", "omegad19*X18||U44", "omegad18*X19||U45"], "osc": {}, "qub": {"0": 2, "1": 2, "2": 2, "3": 2, "4": 2, "5": 2, "6": 2, "7": 2, "8": 2, "9": 2, "10": 2, "11": 2, "12": 2, "13": 2, "14": 2, "15": 2, "16": 2, "17": 2, "18": 2, "19": 2}, "vars": {"jq0q1": 0, "jq10q11": 0, "jq11q12": 0, "jq11q16": 0, "jq12q13": 0, "jq13q14": 0, "jq13q18": 0, "jq15q16": 0, "jq16q17": 0, "jq17q18": 0, "jq18q19": 0, "jq1q2": 0, "jq1q6": 0, "jq2q3": 0, "jq3q4": 0, "jq3q8": 0, "jq5q10": 0, "jq5q6": 0, "jq6q7": 0, "jq7q12": 0, "jq7q8": 0, "jq8q9": 0, "jq9q14": 0, "omegad0": 0, "omegad1": 0, "omegad10": 0, "omegad11": 0, "omegad12": 0, "omegad13": 0, "omegad14": 0, "omegad15": 0, "omegad16": 0, "omegad17": 0, "omegad18": 0, "omegad19": 0, "omegad2": 0, "omegad3": 0, "omegad4": 0, "omegad5": 0, "omegad6": 0, "omegad7": 0, "omegad8": 0, "omegad9": 0, "wq0": 30.518812656662774, "wq1": 31.238229295532093, "wq10": 31.285398043041106, "wq11": 30.926725611566916, "wq12": 30.3733963935667, "wq13": 31.12284550522329, "wq14": 29.847723293111535, "wq15": 29.260290750441424, "wq16": 30.468971343529436, "wq17": 30.06473424474143, "wq18": 30.32505201755447, "wq19": 29.613762827736643, "wq2": 31.86153713573995, "wq3": 30.956376759508178, "wq4": 29.655141246490142, "wq5": 30.002086408102937, "wq6": 30.815575406278047, "wq7": 30.091445635051105, "wq8": 30.52127312814466, "wq9": 29.2747623003134}}, "quantum_volume": 8}
```

Comment:
In all these config JSON files, the name of the identity gate has changed to `'i'`, but the qasm definition within is still
```
gate id a { U(0, 0, 0) a; }
```
do these need to be the same? I.e. change this to 
```
gate i a { U(0, 0, 0) a; }
```

Reply:
no I think here we are just renaming Terra's user-facing gate names (`IGate` and `circuit.i()`). We are not changing QASM nor backend configurations, which are based on QASM.

Reply:
Okay, I reverted the backend configs via b625908. One thing that still changed however are the qobj schemata (e.g. `qiskit/schemas/qobj_schema.json`). These are fine to change, right?

Reply:
Update: keeping the names `sdg`, `tdg` and `id` the schemata do not change.